parsl/monitoring/monitoring.py

```diff
@@ -229,7 +229,7 @@ def send(self, mtype, message):
         self.logger.debug("Sending message {}, {}".format(mtype, message))
         return self._dfk_channel.send_pyobj((mtype, message))
 
-    def __del__(self):
+    def close(self):
```

Comment:
I think there's value in having a __del__ method, for eg when something bad happens and close() isn't called.
We could easily avoid the double call to __del__ in #827 by adding a flag to avoid a second call.

Reply:
fixed. I now use a tag `monitoring_hub_active` to check if it is closed.